modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -13,18 +14,18 @@ class UploadsController < ApplicationController
       skip_after_action :set_csrf_header
 
       FORM_NUMBER_MAP = {
+        '20-10206' => 'vba_20_10206',
+        '20-10207' => 'vba_20_10207',
+        '21-0845' => 'vba_21_0845',
         '21-0966' => 'vba_21_0966',
         '21-0972' => 'vba_21_0972',
-        '21-0845' => 'vba_21_0845',
         '21-10210' => 'vba_21_10210',
         '21-4138' => 'vba_21_4138',
         '21-4142' => 'vba_21_4142',
         '21P-0847' => 'vba_21p_0847',
         '26-4555' => 'vba_26_4555',
         '40-0247' => 'vba_40_0247',
-        '20-10206' => 'vba_20_10206',
-        '40-10007' => 'vba_40_10007',
-        '20-10207' => 'vba_20_10207'
+        '40-10007' => 'vba_40_10007'
       }.freeze
```

Comment:
Alphabetization, to reduce the chance of redundancy and to make it easier to find the form you're looking for.